src/wcs1_0/src/main/java/org/geoserver/wcs/kvp/TimeKvpParser.java

```diff
@@ -0,0 +1,33 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wcs.kvp;
+
+import org.geoserver.config.GeoServer;
+import org.geoserver.ows.kvp.TimeParser;
+import org.geoserver.wcs.WCSInfo;
+
+/**
+ * A {@link TimeKvpParser} picking the max number of values to be parsed from the WMS configuration
```

Comment:
small typo here?
Maybe WCS instead of WMS configuration? (Probably a copy and paste from the WMS's counterpart I found below)

Reply:
Yep, fixing